crates/consensus/src/request.rs

```diff
@@ -0,0 +1,46 @@
+use alloy_eips::{
+    eip6110::DepositRequest,
+    eip7002::WithdrawalRequest,
+    eip7685::{Decodable7685, Eip7685Error, Encodable7685},
+};
+use alloy_rlp::{Decodable, Encodable};
+
+/// Ethereum execution layer requests.
+///
+/// See also [EIP-7685](https://eips.ethereum.org/EIPS/eip-7685).
+pub enum Request {
```

Comment:
derives?

Reply:
yeah sorry i should've marked this as a draft, not done with it